Documentation/Storage-Configuration/Advanced/configuration.md

```diff
@@ -1,20 +1,17 @@
 ---
 title: Configuration
```

Comment:
I'd recommend we merge this content with the Ceph Configuration page. @BlaineEXE sound ok? Let's do it as a follow-up PR though.